sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1664,6 +1664,30 @@ def boundary_conditions(self):
         """
         return self._boundary_conditions
 
+
+    def polar_moment(self):
+        """
+        This function calculates the Polar moment of Inertia of the beam about the x axis.
```

Comment:
Change to "Returns the polar moment of area of the beam about the X axis with respect to the centroid."